kpi/views/v2/asset_permission_assignment.py

```diff
@@ -232,7 +232,15 @@ def clone(self, request, *args, **kwargs):
     def destroy(self, request, *args, **kwargs):
         object_permission = self.get_object()
         user = object_permission.user
-        if user.pk == self.asset.owner_id:
+        # If the user is not the owner of the asset, but trying to delete the
+        # owner's permissions, raise permission denied error. However, if it is
+        # the owner of the asset, they should also be prevented from deleting
```

Comment:
since i hear you're interested in grammar ;) i'd suggest using the singular `they` pronoun in both places: "However, if they are…, they should also"

Reply:
:joy: where did you hear that? Thanks for the suggestion though 💪

Reply:
Just recalling that "lifetime" non-profit [subscription](https://chat.kobotoolbox.org/#narrow/stream/4-KoBo-Dev/topic/Grammarly/near/11418) you got us :wink: